src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,55 @@
+package duke;
+
+import tasks.*;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Storage {
+    private final String filePath;
+
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+    public List<Task> loadFile() throws Exception {
+        List<Task> initList = new ArrayList<>();
```

Comment:
Perhaps we can rename this Collection variable to be `tasks` as it is a collection of Tasks?